blue_level/pizzabot.js

```diff
@@ -0,0 +1,54 @@
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
+
+const validateOrderName = (orderName) => {
+  if (
+    orderName === "vegetarian" ||
+    orderName === "pepperoni" ||
+    orderName === "hawaiian"
+  ) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+const calculateTotalCost = (orderQty, pizzaPrice) => {
+  return orderQty * pizzaPrice;
+};
```

Comment:
Very nice :) Another tip from my side - since we have only one line inside `calculateTotalCost` function, we can write it in "arrow function shorthand" :

`const calculateTotalCost = (orderQty, pizzaPrice) => orderQty * pizzaPrice;`

We can do that because in arrow functions we can omit curly brackets (`{}`) and `return` keyword if we only have one line of code inside.
For further reference on that subject, please check out https://www.w3schools.com/js/js_arrow_function.asp

Reply:
:D

Reply:
For some reason VS Code keeps flagging the version without curly brackets as an error. :(

So when the code looks like this: `const calculateTotalCost = (orderQty, pizzaPrice) => return orderQty * pizzaPrice;`

Reply:
done!